### ISSUE TITLE
add services.restic ?

### ISSUE THREAD

**Issue by sedlund** [nix-community/home-manager#5009]
### Description

Is it possible to include the NixOS restic service module?  

https://github.com/NixOS/nixpkgs/blob/nixos-unstable/nixos/modules/services/backup/restic.nix

Would be useful on machines that I only use home-manager on.

**Comment by sudoforge** [comment]
I intend to work on this some time later this year when my priorities and schedule allow for it.

**Comment by kmf** [comment]
I'm here for this

**Comment by ttrssreal** [comment]
hi I might work on this

**Comment by ttrssreal** [comment]
Proper support for the rclone backend seems to be [blocked on this](https://github.com/nix-community/home-manager/issues/2703)

**Comment by kuflierl** [comment]
This seems unblocked now, any interest here?

**Comment by ttrssreal** [comment]
Yep, working on it now, should be just the integration tests that need finishing
